src/main/java/io/phdata/streamliner/schemadefiner/mapper/SnowflakeMapper.java

```diff
@@ -31,13 +36,35 @@
 public class SnowflakeMapper {
   private static final Logger log = LoggerFactory.getLogger(SnowflakeMapper.class);
 
-  public static List<TableDefinition> mapSchemaCrawlerTables(
-      List<Table> tables, List<UserDefinedTable> userDefinedTables, Snowflake destination) {
+  private final Jdbc source;
+  private final Snowflake destination;
+  private final String jdbcPassword;
+  private final AttributeRetriever attributeRetriever;
+  private final Supplier<Connection> conSupplier =
+      new Supplier<Connection>() {
+        @Override
+        public Connection get() {
+          return StreamlinerUtil.getConnection(source.getUrl(), source.getUsername(), jdbcPassword);
+        }
+      };
+
+  public SnowflakeMapper(
```

Comment:
This should be in a HiveMapper right? It seems like we are trying to jam Hive stuff into something that was built for Snowflake

Reply:
I  think the naming is given based on the destination. Since Snowflake is the destination, all the tables, and columns related operations are added in this class for all sources.

Reply:
I see

Reply:
I see